File: AC/ABC-TDD/166/b.py
n, k = map(int, input().split())
sunuke = [False]*n

for i in range(k):
    d = int(input())
    arr = list(map(int, input().split()))
    for j in range(d):
        sunuke[arr[j]-1] = True

cnt = 0
for v in sunuke:
    if not v:
        cnt += 1
print(cnt)

File: AC/ABC-TDD/166/b.go
package main

import "fmt"

func main() {
	var n, k, d, tmp, cnt int
	fmt.Scan(&n, &k)
	sunuke := make([]bool, n)
	for i := 0; i < k; i++ {
		fmt.Scan(&d)
		for j := 0; j < d; j++ {
			fmt.Scan(&tmp)
			sunuke[tmp-1] = true
		}
	}
	for _, v := range sunuke {
		if !v {
			cnt++
		}
	}
	fmt.Println(cnt)
}
